examples/animation/animate_decay.py

```diff
@@ -32,6 +32,6 @@ def run(data):
 
     return line,
 
-ani = animation.FuncAnimation(fig, run, data_gen, blit=True, interval=10,
+ani = animation.FuncAnimation(fig, run, data_gen, blit=False, interval=10,
```

Comment:
ah, ok.  If you move the mouse in and out it triggers a full re-draw (at least on qt) which updates them so I thought it was always updating correctly.

There probably should be some logic someplace in the animation that if any has change to do a full-redraw.

This actually ties back into what @efiring and I were talking about to make pyplot behave better as well as it wouldn't be too hard to co-op that system to allow non-animated artists to make the axes as 'dirty' which would trigger a full-redraw.